.jshintrc

```diff
@@ -27,7 +27,7 @@
     "unused"          : true,      // Warns when you define and never use your variables.
     "strict"          : false,     // Requires all functions to run in ECMAScript 5's strict mode.
     "trailing"        : true,      // Makes it an error to leave a trailing whitespace in your code.
-    "maxlen"          : 120,       // Lets you set the maximum length of a line.
+    //"maxlen"        : 120,       // Lets you set the maximum length of a line.
```

Comment:
Any reason why we don't just delete maxlen? Is it worth a comment about why jshint can no longer check max line length?